cypress/support/step-definitions/text-area-steps.js

```diff
@@ -97,11 +97,11 @@ Then('Textarea inputWidth is set to {string}', (width) => {
 });
 
 Then('Textarea component is labelInline', () => {
-  label().should('have.css', TEXT_ALIGN, 'left');
+  label().parent().should('have.css', TEXT_ALIGN, 'flex-start');
```

Comment:
The same here about transformation `flex-start` into a const

Reply:
fixed